datafusion/physical-expr/src/array_expressions.rs

```diff
@@ -2514,6 +2520,67 @@ mod tests {
             .is_null(0));
     }
 
+    #[test]
+    fn test_array_range() {
```

Comment:
remove test here

Reply:
@jayzhan211  and @Weijun-H  I think what is happening is that people follow the model of the existing code -- so as long as there are tests in array_expressions (rather than .slt) that is what people will model.

What do you think about migrating (as part of another PR) all the existing tests to sqllogictests -- this would likely result in all subsequent PRs following the sqllogictests model

Reply:
I endorse this proposal as it promises to enhance convenience in directing individuals to incorporate new tests, thereby minimizing potential misunderstandings.

Reply:
@alamb @Weijun-H 
There is one problem actually, function like `align_array_dimensions` we have a unit test for it but it is not possible to port to slt file. We should either not test it directly but rely on array function test that call it or port it to datafusion::common::utils, and test it there. I think the latter is better so we can check the coverage more easily.

btw, should we introduce array_utils in common. I think it will grow larger and larger quickly.